lib/pipeline/RequestHandler.js

```diff
@@ -35,8 +35,12 @@ class RequestHandler extends Duplex {
       metrics.histogram('limitd.database.took', took, tags);
 
       if (err) {
-        if (err.message.indexOf('undefined bucket type') > -1 && !request.skipResponse) {
-          this._queueResponse(request, { error: { type: 'UNKNOWN_BUCKET_TYPE' } });
+        if (err.message.indexOf('undefined bucket type') > -1) {
+          if (!request.skipResponse) {
```

Comment:
minor: I know this is picky, since we have an "else" here, can we put this in a positive first (so we don't have the inverse order of branches)